Regression/WarpX-tests.ini

```diff
@@ -624,7 +624,7 @@ analysisOutputImage = Langmuir_multi_2d_MR.png
 [Langmuir_multi_2d_MR_psatd]
 buildDir = .
 inputFile = Examples/Tests/Langmuir/inputs_2d_multi_rt
-runtime_params = algo.maxwell_solver = psatd  warpx.use_filter = 1  amr.max_level = 1  amr.ref_ratio = 4  warpx.fine_tag_lo = -10.e-6 -10.e-6  warpx.fine_tag_hi = 10.e-6 10.e-6  diag1.electrons.variables = w ux uy uz  diag1.positrons.variables = w ux uy uz
+runtime_params = algo.maxwell_solver = psatd  warpx.use_filter = 1  amr.max_level = 1  amr.ref_ratio = 4  warpx.fine_tag_lo = -10.e-6 -10.e-6  warpx.fine_tag_hi = 10.e-6 10.e-6  warpx.pml_ncell = 36  warpx.pml_delta = 36  diag1.electrons.variables = w ux uy uz  diag1.positrons.variables = w ux uy uz
```

Comment:
Here I had to increase `pml_ncell` and `pml_delta` from 10 (default) to 36. The check was otherwise failing and my understanding is that this was happening because we have a mesh refinement ratio of 4, 8 guard cells, and need to make sure that when we construct the coarse BoxArray for the PML,
https://github.com/ECP-WarpX/WarpX/blob/2476d50439e39843d75720c3f857ad05c4f89e23/Source/BoundaryConditions/PML.cpp#L776-L781
the ratios of `ncell` and `delta` over the refinement ratio are still larger than the number of guard cells.

This seems to require using a lot of cells for the PML grid when using mesh refinement with high refinement ratios. So I'm not sure that I'm fixing this correctly, it would need to be double checked. 

Also, we might not need to increase both `pml_ncell` and `pml_delta`, `pml_ncell` might be sufficient (but I'm not sure about what happens algorithmically if `pml_delta` is smaller than `pml_ncell`, e.g., what kinds of consequences this has on the absorption rate of the PML).

Reply:
@prkkumar These are the PR and the comment on guard cells v. PML width with MR that we talked about before, just for reference. Your feedback on this is welcome as well, since you're trying out similar things on your LWFA tests with MR.